pkg/decoder/tagsl/v1/port105.go

```diff
@@ -0,0 +1,34 @@
+package tagsl
+
+import "time"
+
+// +-------+------+-------------------------------------------+-----------+
+// | Byte  | Size | Description                               | Format    |
+// +-------+------+-------------------------------------------+-----------+
+// | 0     | 2    | Buffer level                              | uint8     |
```

Comment:
Byte column has some errors in there, also for the other ports e.g. buffer level 0-1, unix timestamp 2-5 etc.